x/evm/types/context.go

```diff
@@ -10,13 +10,25 @@ import (
 	"github.com/sei-protocol/sei-chain/x/evm/types/ethtx"
 )
 
+type Version int
+
+const (
+	Frontier Version = iota
+	Homestead
+	EIP155
+	Berlin
+	London
+	Cancun
+)
```

Comment:
Just for my education: can you explain why these are the only versions we are including? Also was wondering why we don't have the latest fork which I believe is Shanghai, and are including Cancun which is a future fork.

Reply:
this is for deciding Signer version so only these versions are relevant. Changed the type name to reflect that. Also trimmed unnecessary version since we will start from signer version London (overall version Shanghai)